modules/ROOT/pages/availability_scaling/index.adoc

```diff
@@ -0,0 +1,140 @@
+= Availability and Scalability
+:toc: right
+:toclevels: 2
+
+:gartner_complexity_url: https://blogs.gartner.com/ephraim-baron/how-systems-complexity-reduces-uptime/
+:what_is_kubernetes_url: https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/
+
+:description: There are many ways to achieve availability and scalability, but certain aspects depend on each other and decisions have to be considered carefully to optimize the outcome. This guide gives a brief overview about running a modern, microservices-based software like oCIS as a single or distributed instance.
+
+== Introduction
+
+{description}
+
+Because oCIS is by design enabled to run as a distributed service and not a static monolithic service block, you can achieve availability _and_ scalability with well made decisions. Due to this design, decisions made can always be revised and adapted to new requirements.
+
+== Terminology
+
+* High availability is a method that groups servers that support applications or services that can be reliably used with a minimum amount of downtime.
+
+* Scalability means adding more instances.
+** Horizontal scalability, which is also known as *scale-out*, can be performed by adding more hardware resources like physical or virtual servers.
+** Vertical Scalability, also known as *scale-up*, can be performed by getting more powerful hardware that can either outperform the replaced one and/or run more services.
+
+* Virtualization is the ability to abstract hardware, software or a deployment.
+** Virtualized hardware like VMWare, KVM, HyperV, VirtualBox etc.
+** Virtualized Linux operating systems in Docker containers.
+** Virtualization of a deployment, e.g. with Kubernetes.
+
+== Supporting Questions
+
+The following questions may help in the decision:
+
+* What is the goal?
+* What do you want to protect against?
+* What measures are needed to achieve this protection?
+* How do you want to scale?
+* What is the level of complexity coming along with the measures taken like implementation, maintenance, ability to adopt to dynamic requirements, etc.?
+* What are the costs for implementation and runtime?
+* Will the measures taken result in the goal you have set?
+
+This questionnaire is not static and more rounds may be needed to refine each item. With every round the result will become better and clearer. Tasks and actions can be taken and responsibilities can be defined. 
+
+== Availability
+
+Traditionally, availability concepts often focus on monolithic hardware and software components. This changes when you take a services-centered view. While a service may be up and running, it might not respond accordingly to requests because of scaling factors. This means for clients, the driver for the measures taken it is not "available". Using a sophisticated hardware setup can be one way to solve this. But it usually does not or only partially solve the scalability issues.
```

Comment:
```suggestion
Traditionally, availability concepts often focus on monolithic hardware and software components. This changes when you take a services-centered view. While a service may be up and running, it might not respond accordingly to requests because of scaling factors. This means for clients, the driver for the measures taken is not just "available". Using a sophisticated hardware setup can be one way to solve this. But it usually does not or only partially solves the scalability issues.
```